From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@changeworld__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmcmurray commented:
#sign-off

Commit:
Introducing POA for linux.
Added links for better visibility of article.

adegeo committed:
fixed sizes

Commit:
Added article for service fabric patch orchestration app linux.

Commit:
Self-review iteration 1

Updated acrolinx changes.
Fixed typos and copy paste errors.

Commit:
testing new format changes

Commit:
writing

Commit:
Fixed acrolinx issues for both windows and linux articles

Commit:
event source -> eventsource
Azure specific acrolinx violations were fixed (these won't show on web interface).

mimig1 committed:
Cosmos DB: Added new access control topic

mimig1 committed:
fixed folder location

Commit:
addressing TFS1179622 and further clarifications

Commit:
update for policy checks

cforbe committed:
Merge pull request #2 from MicrosoftDocs/master

Merge

Commit:
luis02

Commit:
Added reminders for access polices pane

to save and close

Commit:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

Commit:
updated the date

Commit:
Write doc on operationalization

Includes information on real-time scoring and the ReadJSONLiteral transformation

shuyu42 committed:
Update data-prep-appendix9-sample-destination-connections-python.md

shuyu42 committed:
Update data-prep-appendix8-sample-source-connections-python.md

Commit:
Include operationalization doc in table of contents

cforbe committed:
Merge pull request #5 from shuyums2/patch-2

Update data-prep-appendix8-sample-source-connections-python.md

cforbe committed:
Merge pull request #4 from shuyums2/patch-1

Update data-prep-appendix9-sample-destination-connections-python.md

cforbe committed:
Merge pull request #3 from glasperfan/patch-1

Write doc on data prep operationalization

cforbe committed:
Merge pull request #6 from glasperfan/patch-2

Include operationalization doc in table of contents

rongduan-zhu committed:
Removed use of wrangle

mattdmorgan committed:
Fix file paths to Python

Commit:
Update set data type info for data source wizard

mattdmorgan committed:
Merge pull request #1 from mattdmorgan/mattdmorgan-patch-1

Fix file paths to Python

cforbe committed:
Merge pull request #7 from rongduan-zhu/patch-4

Removed use of wrangle

cforbe committed:
Merge pull request #8 from glasperfan/patch-3

Update set data type info for data source wizard

cforbe committed:
Merge pull request #9 from mattdmorgan/master

Fix paths to Python

ajaycode committed:
Added reference to NPM

ajaycode committed:
Updated with latest images and removed references to preview.

ajaycode committed:
Removed reference to image 6.png

ajaycode committed:
Merge pull request #3 from ajaycode/ajaycode-patch-2

Removed reference to image 6.png

tdykstra committed:
add memory usage note

tdykstra committed:
typo

tdykstra committed:
smll change to trigger publication

tdykstra committed:
use queues include

tdykstra committed:
paulbatum feedback

rayne-wiselman committed:
updating file names for hyper-v-vmm scenario

rayne-wiselman committed:
Update hyper-v-vmm-networking.md

rayne-wiselman committed:
Update hyper-v-vmm-performance-results.md

rayne-wiselman committed:
Update hyper-v-vmm-architecture.md

rayne-wiselman committed:
Update hyper-v-vmm-disaster-recovery.md

rayne-wiselman committed:
Update hyper-v-vmm-failover-failback.md

rayne-wiselman committed:
Update hyper-v-vmm-networking.md

rayne-wiselman committed:
adding missing graphic files

rayne-wiselman committed:
Update hyper-v-vmm-powershell-resource-manager.md

rayne-wiselman committed:
Update hyper-v-vmm-test-failover.md

rayne-wiselman committed:
Update .openpublishing.redirection.json

rayne-wiselman committed:
Update hyper-v-vmm-performance-results.md

rayne-wiselman committed:
Update hyper-v-vmm-test-failover.md

rayne-wiselman committed:
Update hyper-v-vmm-performance-results.md

rayne-wiselman committed:
updating link

rayne-wiselman committed:
Update site-recovery-active-directory.md

rayne-wiselman committed:
Update hyper-v-vmm-architecture.md

rayne-wiselman committed:
Update hyper-v-vmm-network-mapping.md

rayne-wiselman committed:
Update hyper-v-vmm-recovery-script.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hivemvc

rayne-wiselman committed:
Update hyper-v-vmm-test-failover.md

Commit:
writing

Commit:
updates for Hive

Commit:
fixing H2

alexwun committed:
Update default service section

Commit:
fixed enroll device selectors to include C# and added include to enroll device C# articles

Commit:
fixed x509 selector -- added iot-dps to path for c#

Commit:
making scripts prompt for user input and/or programmatically get it

ggailey777 committed:
Updated the metadata for the control

Commit:
caintro02

MGoedtel committed:
updated article

MRayermannMSFT committed:
Add information regarding Linux dependencies for Storage Explorer

MRayermannMSFT committed:
GCC in getting started

cforbe committed:
Fixes to documents

Commit:
caintro02

sanjeev3 committed:
Update facebook-post-moderation.md

Commit:
re-ordered x.509 create device quickstarts- python AFTER node

sanjeev3 committed:
Update facebook-post-moderation.md

sanjeev3 committed:
Update facebook-post-moderation.md

PRMerger20 committed:
Merge pull request #33331 from sanjeev3/master

Added a note and minor edits to one line.

rmca14 committed:
Merge pull request #33301 from alexwun/master

Update default service section

Commit:
delfacebook01

Lisaco88 committed:
Merge pull request #33321 from MarkusVi/caintro02

caintro02

rmca14 committed:
Merge pull request #33334 from MarkusVi/delfacebook01

delfacebook01

Juliako committed:
updated

kgremban committed:
Adding breadcrumb for LBS

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dps-selector-fix
sync with upstream before push

MGoedtel committed:
updated intro paragraph

adegeo committed:
Cleaning up some of the diagnostics files

Commit:
luis02

Lisaco88 committed:
Merge pull request #33281 from cforbe/master

Updating Data Prep Docs

adegeo committed:
Fixed link to deleted file

Commit:
luis02

ArLucaID committed:
Updated Linux List

rmca14 committed:
Merge pull request #33337 from kgremban/feb13-lbsbreadcrumb

Adding breadcrumb for LBS

ArLucaID committed:
Update msi-known-issues.md

PRMerger19 committed:
Merge pull request #33342 from arluca/patch-4

MSI - Update Linux List of supported OS's

Juliako committed:
Update automation-manage-api-management.md

Juliako committed:
Update api-management-template-data-model-reference.md

Commit:
harshini02

Juliako committed:
Update api-management-sample-cache-by-key.md

Juliako committed:
Update automation-manage-api-management.md

MRayermannMSFT committed:
Fix some style issues.

Commit:
harshini04

Juliako committed:
Update api-management-sample-send-request.md

MRayermannMSFT committed:
Remove a space

PRMerger20 committed:
Merge pull request #33344 from Juliako/patch-2

Update api-management-template-data-model-reference.md

Commit:
Merge pull request #33343 from Juliako/patch-1

Update automation-manage-api-management.md

Commit:
harshini04

PRMerger12 committed:
Merge pull request #31583 from Thraka/cs-sizes

fixed sizes

PRMerger12 committed:
Merge pull request #33349 from MarkusVi/harshini04

harshini04

PRMerger13 committed:
Merge pull request #33345 from MarkusVi/harshini02

harshini02

mimig1 committed:
Incorporated Armando's feedback

Juliako committed:
Update api-management-sample-cache-by-key.md

Lisaco88 committed:
Merge pull request #33304 from JimacoMS2/dps-selector-fix

fixed enroll device selectors to include C# and added include to enro…

Juliako committed:
Update api-management-sample-send-request.md

Juliako committed:
Update api-management-sample-cache-by-key.md

Juliako committed:
Update api-management-sample-cache-by-key.md

PRMerger20 committed:
Merge pull request #33350 from Juliako/patch-4

Update api-management-sample-send-request.md

Lisaco88 committed:
Merge pull request #33323 from MGoedtel/UpdateOMSG213

updated article

Lisaco88 committed:
Merge pull request #33336 from Juliako/APIMUpdate

updated

PRMerger20 committed:
Merge pull request #33347 from Juliako/patch-3

Update api-management-sample-cache-by-key.md

v-nagta committed:
updated the missing icons

Juliako committed:
updated

Juliako committed:
Update api-management-sample-send-request.md

Juliako committed:
Update api-management-sample-send-request.md

PRMerger16 committed:
Merge pull request #33356 from Juliako/UpdateAPIM2

updated

PRMerger15 committed:
Merge pull request #33346 from MRayermannMSFT/add-linux-dependencies-information-for-stg-exp

Add linux dependencies information for stg exp

adegeo committed:
Added redirects

Commit:
Added the download link for Patch Orchestration Application linux.

PRMerger13 committed:
Merge pull request #33316 from ggailey777/patch-10

Updated the metadata for the control

Commit:
Merge remote-tracking branch 'upstream/master'

dominicbetts committed:
Fix location of MXChip link

dominicbetts committed:
Add StackOverflow link

Commit:
Modifying the TOC to include windows and linux content under the heading "Patch Cluster nodes"

dominicbetts committed:
Add C video

Commit:
Merge pull request #33360 from dominicbetts/hub-c-video

Add C video

MGoedtel committed:
minor edits to improve article

PRMerger18 committed:
Merge pull request #33361 from MGoedtel/UpdateOMSG214

minor edits to improve article

Commit:
Freshness updates.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aas0214

Commit:
renumbering

Commit:
Acrolinx fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hivemvc

daveirwin1 committed:
Updated VM Extension version in samples

PRMerger20 committed:
Merge pull request #33363 from daveirwin1/master

Updated VM Extension version in samples

Lisaco88 committed:
Merge pull request #33339 from Thraka/cs-deletevs

Delete obsolete visual studio articles

rjagiewich committed:
Merge pull request #33265 from rayne-wiselman/rayne-february13

updating file names for hyper-v-vmm scenario

rjagiewich committed:
Merge pull request #33358 from dominicbetts/suite-fix-toc

Fix location of MXChip link

Lisaco88 committed:
Merge pull request #31939 from novino/master

Introducing Patch Orchestration Application for Linux

Lisaco88 committed:
Merge pull request #32966 from mimig1/02062018accesscontrol

Cosmos DB: Added new access control topic

rjagiewich committed:
Merge pull request #32264 from Blackmist/hivemvc

testing new format changes

cherylmc committed:
graphics

Lisaco88 committed:
Merge pull request #32989 from clemensv/tfs1179622

addressing TFS1179622 and further clarifications

cherylmc committed:
update

Commit:
luis02

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

bandersmsft committed:
cost mgt - updated faq for currency

rjagiewich committed:
Merge pull request #33209 from tdykstra/blob

add memory usage note to blob binding ref

PRMerger12 committed:
Merge pull request #33131 from ChackDan/master

Master

Lisaco88 committed:
Merge pull request #33115 from MarkusVi/luis02

luis02

rjagiewich committed:
Merge pull request #33355 from v-nagta/iconsv2

updated the missing icons

Commit:
Merge pull request #33372 from bandersmsft/master

cost mgt - updated faq for currency

v-alje committed:
Merge pull request #33371 from MicrosoftDocs/FromPublicRepo

2/14 AM Publish

PRMerger10 committed:
Merge pull request #33180 from ajaycode/patch-7

Added reference to NPM

Commit:
Merge pull request #33187 from ajaycode/patch-8

Updated with latest images and removed references to preview.

rjagiewich committed:
Merge pull request #33362 from Minewiskan/aas0214

Aas0214

rjagiewich committed:
Merge pull request #33369 from cherylmc/npm

graphics
